resources/views/layouts/master.blade.php

```diff
@@ -10,15 +10,15 @@
 	<title>CuapCuap &middot; @yield('title')</title>
 
 	@section('external-resource')
-		<link rel="stylesheet" href="http://cdn.everything.io/kickstart/3.x/css/kickstart.min.css" />
-		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+			<link rel="stylesheet" href="{!! asset('css/kickstart.min.css) !!}">
+			<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
 	@show
 
 	<style>
 		@section('internal-css')
 			@font-face {
 			    font-family: 'DroidSerif-Regular';
-			    src: url('/fonts/DroidSerif-Regular.ttf');
```

Comment:
if the / is removed, it won't always point to public/fonts, instead, if the current URL is .../thread/x it will point to public/thread/fonts